repository_service_tuf_api/bootstrap.py

```diff
@@ -228,16 +210,6 @@ def post_bootstrap(payload):
             role_settings.expiration,
             settings_repository,
         )
-        save_settings(
-            f"{rolename}_THRESHOLD",
-            role_settings.threshold,
-            settings_repository,
-        )
-        save_settings(
-            f"{rolename}_NUM_KEYS",
-            role_settings.num_of_keys,
-            settings_repository,
-        )
```

Comment:
Those settings are also required to be in the Redis for Root role.
In the future, in case of key rotation and changing the number of the keys or the threshold all the workers need to have it as live information from the Redis.